app/views/motorbikes/show.html.erb

```diff
@@ -1 +1 @@
-<div id="root"></div>
+<div id="root" class="container"></div>
```

Comment:
You already mentioned `class="container"` in the application.html.erb file, at the end the out will be :
```
<div class="container">
   <div id="root" class="container"></div>
</div> 
```